torchvision/datasets/utils.py

```diff
@@ -18,6 +20,20 @@ def check_integrity(fpath, md5):
     return True
 
 
+def reporthook(count, block_size, total_size):
+    global start_time
```

Comment:
I don't think `start_time` is defined anywhere in this file.

Let's remove this functional altogether for now, and maybe send another PR adding it if necessary?